### PR TITLE
Move API Playground to components but do not expose the values yet

### PR DESCRIPTION
**Pull request by tobias-mintlify**
Breaks up the API Playground into granular components and adds each of them to Storybook.

I'll make another PR tomorrow that makes the API Playground functional. Test this PR by viewing the stories for the components that make up the playground.

### CODE REVIEW

File: src/Api/ApiPlayground.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  //   param: string,
+  //   value: ApiInputValue,
+  //   path: string[]
+  // ) => {
+  //   const newParamGroup = {
+  //     ...inputData[paramGroup],
+  //     ...set(inputData[paramGroup], [...path, param], value),
+  //   };
+  //   setInputData({ ...inputData, [paramGroup]: newParamGroup });
+  // };
```
Comment (hahnbeelee) [inline src/Api/ApiPlayground.tsx:64]: What is this?

File: src/Api/ApiPlayground.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+// <div className="py-3 px-3 max-h-60 whitespace-pre overflow-scroll border-t border-slate-200 dark:border-slate-700  dark:text-slate-300 font-mono text-xs leading-5">
+//           <span
+//             className="language-json max-h-72 overflow-scroll"
+//             dangerouslySetInnerHTML={{
+//               __html: apiResponse,
+//             }}
+//           />
+//         </div>
```
Comment (hahnbeelee) [inline src/Api/ApiPlayground.tsx:152]: ?

**Review by hahnbeelee — APPROVED** [review]
lgtm

File: src/Api/ApiPlayground.tsx (hunk 1 above)
Comment (tobias-mintlify) [reply src/Api/ApiPlayground.tsx:64]: I'm working on using these today! I'm not publishing to npm yet so I'm going to merge for now.

**Pull request merged**